README.md

```diff
@@ -46,7 +48,16 @@ pipe([1, 2, 3], stream)
 
 ## API
 
-### `const muxer = new Mplex([options])`
+### `const factory = new Mplex([options])`
+
+Creates a factory that can be used to create new muxers.
+
+`options` is an optional `Object` that may have the following properties:
+
+* `maxMsgSize` - a number that defines how large mplex data messages can be in bytes, if messages are larger than this they will be sent as multiple messages (default: 1048576 - e.g. 1MB)
+* `maxStreamsPerConnection` - a number that defines how many streams are allowed per connection (default: 1024)
```

Comment:
The default values listed in these docs can get out of date.  Can/should we link to the authoritative source?

Reply:
Sure, generating docs from the interface might be a better approach here.